src/main/java/frc/robot/commands/ToggleIntakePistonCommand.java

```diff
@@ -0,0 +1,27 @@
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.IntakeSubsystem;
+
+public class ToggleIntakePistonCommand extends CommandBase {
+    private IntakeSubsystem intakeSubsystem;
+
+    public ToggleIntakePistonCommand(IntakeSubsystem intakeSubsystem) {
+        this.intakeSubsystem = intakeSubsystem;
+    }
+
+    @Override
+    public void initialize() {
+        this.intakeSubsystem.extend();
```

Comment:
The command is called toggle but this only extends.

Reply:
Good point, the command actually just retracts the intake when it ends, so it functions as an argument to `toggleWhenPressed`. It's a little weird but it works nicely when tested.